onto_tool/command_line.py

```diff
@@ -198,6 +198,9 @@ def define_graphic_parser(subparsers):
         " graphing an endpoint, or 'Gist' if graphing local files.")
     graphic_parser.add_argument('ontology', nargs="*", default=[],
                                 help="Ontology file, directory or name pattern")
+    graphic_parser.add_argument("--show-bnode-subjects", action="store_true",
+                                help="Use triples with blank nodes in the subject to generate "
```

Comment:
I would modify the description to something like _Include blank node subjects when generating a graph._